tools/make-release

```diff
@@ -29,17 +29,23 @@ cd /tmp
 [ -e ubuntu-advantage-client ] && rm -rf ubuntu-advantage-client
 git clone git@github.com:CanonicalLtd/ubuntu-advantage-client.git
 cd ubuntu-advantage-client
-VERSION=`dpkg-parsechangelog -S Version`
-sed -i "s/ubuntu-advantage-tools (${VERSION}) [[:alpha:]]\+;/ubuntu-advantage-tools (${VERSION}) ${SERIES};/" debian/changelog
+CHANGELOG_VERSION=`dpkg-parsechangelog -S Version`
+GIT_VERSION=`git describe --long`
+RELEASE_NUMBER=`distro-info --series ${SERIES} -r`
```

Comment:
I tend to like $(), as you've done in the make-tarball script, over `` backticks for shell commands.